### ISSUE TITLE
google pay sdk

### ISSUE THREAD

**Issue by bsde1234** [chrisdiana/simplestore#37]
Is it possible to add google pay web api?

**Comment by peterochieng** [comment]
With a little tweaking here and there it's possible

**Comment by bsde1234** [comment]
@peterochieng what edits are needed. Can you post the edits

**Comment by chrisdiana** [comment]
Yeah this is definitely on my radar as well. Thanks for the suggestion! Should be too bad to add this as an additional plugin.

**Comment by bsde1234** [comment]
I found a project add login [here](https://github.com/mitul45/expense-manager), but google temporarly [stops it](https://github.com/mitul45/expense-manager/issues/18). Is there any other way to add a login to this project.

**Comment by bsde1234** [comment]
I have following questions...
1. In India we use upi id instead of cards in Google pay. So if I get grant total from cart page I can add payment button like `upi://pay?pa=my_upi_id@upi&pn=payee&am=total_amount_here&tn=Test_Transaction`.
2. Currently if I pay using PayPal will the page return back to project and give receipt?
